engine/render/src/render/render_script.cpp

```diff
@@ -1309,21 +1351,16 @@ namespace dmRender
     int RenderScript_SetRenderTargetSize(lua_State* L)
     {
         RenderScriptInstance* i = RenderScriptInstance_Check(L);
-        (void)i;
-        dmGraphics::HRenderTarget render_target = 0x0;
-
-        if (lua_isnumber(L, 1))
-        {
-            render_target = (dmGraphics::HRenderTarget) lua_tonumber(L, 1);
-            uint32_t width = luaL_checknumber(L, 2);
-            uint32_t height = luaL_checknumber(L, 3);
-            dmGraphics::SetRenderTargetSize(render_target, width, height);
-            return 0;
-        }
-        else
+        if (!lua_isnumber(L, 1))
         {
             return luaL_error(L, "Expected render target as the second argument to %s.set_render_target_size.", RENDER_SCRIPT_LIB_NAME);
         }
+
+        dmGraphics::HRenderTarget render_target = (dmGraphics::HRenderTarget) CheckAssetHandle(L, 1, i->m_RenderContext->m_GraphicsContext, dmGraphics::ASSET_TYPE_RENDER_TARGET);
+        uint32_t width = luaL_checknumber(L, 2);
+        uint32_t height = luaL_checknumber(L, 3);
```

Comment:
Integers?

Reply:
Yes (think this is also old moved code) I'll change it